src/Collection/Collection.service.spec.ts

```diff
@@ -0,0 +1,92 @@
+import { v4 as uuidv4 } from 'uuid'
+import { Collection } from './Collection.model'
+import { CollectionService } from './Collection.service'
+import { wallet } from '../../spec/utils'
+
+jest.mock('./Collection.model')
+
+describe.only('Collection service', () => {
+  describe('#isLocked', () => {
+    const collectionAttributes = {
+      id: uuidv4(),
+      name: 'Test',
+      eth_address: wallet.address,
+      salt: '',
+      contract_address: '0x2a8fd99c19271f4f04b1b7b9c4f7cf264b626edb',
+      is_published: false,
+      is_approved: false,
+      minters: [],
+      managers: [],
+      forum_link: null,
+      lock: null,
+      reviewed_at: new Date(),
+      created_at: new Date(),
+      updated_at: new Date(),
+    }
+
+    const service = new CollectionService()
+    const findOneMock = Collection.findOne as jest.Mock
```

Comment:
What if we do a mockRestore of `findOneMock` on a `beforeEach` to be sure that it gets cleaned after every execution?